src/test/java/teammates/test/pageobjects/InstructorStudentRecordsPage.java

```diff
@@ -69,6 +74,23 @@ private boolean areAllRecordPanelBodiesVisibilityEquals(boolean isVisible) {
         return true;
     }
 
+    public void closeEditFeedbackResponseCommentForm(String commentIdSuffix) {
+        WebElement editResponseForm = browser.driver.findElement(By.id("responseCommentEditForm" + commentIdSuffix));
+        WebElement cancelButton = editResponseForm.findElement(By.cssSelector("input[type='button']"));
+        click(cancelButton);
+    }
+
+    public void editFeedbackResponseComment(String commentIdSuffix, String newCommentText) {
+        executeScript("scroll(0,300)");
+        WebElement commentRow = waitForElementPresence(By.id("responseCommentRow" + commentIdSuffix));
+        click(commentRow.findElements(By.tagName("a")).get(1));
+
+        WebElement commentEditForm = browser.driver.findElement(By.id("responseCommentEditForm" + commentIdSuffix));
+        fillRichTextEditor("responsecommenttext" + commentIdSuffix, newCommentText);
+        click(commentEditForm.findElement(By.className("col-sm-offset-5")).findElement(By.tagName("a")));
+        ThreadHelper.waitFor(1000);
```

Comment:
Could use `waitForElementToDisappear()` to wait for an element to disappear instead (such as the input box). However, considering this was copied over from `InstructorFeedbackResultsPage`, it might be best to get rid of `ThreadHelper.waitFor()` in a separate issue (eg. in #3317, or a new issue) to ensure we deal with both in a consistent manner.

No action should be necessary here.